### PR TITLE
Component header brand

### LINKED ISSUE #40: Component: Header / Brand
Brand component.

### PR DESCRIPTION
**Pull request by TeddTech**
resolves #40

### CODE REVIEW

**Comment by TeddTech** [comment]
> @TeddTech I'd like you to get into the habit of running the eslint command on each branch before you open a pull request and making sure that any errors are fixed.
> 
> `npm run lint` shows me a few errors in the `Header.vue` and `Brand.vue` components. Please address these and then request my review again.

Okay, will do

File: components/Brand.vue [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<template>
+	<div class="brand">
+		<img src="~/static/img/We Count logos_colour and bw-01.png">
+		<h1> We Count </h1>
```
Comment (greatislander) [inline components/Brand.vue]: Remove `<h1>` tag.

File: components/Brand.vue [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<template>
+	<div class="brand">
+		<img src="~/static/img/We Count logos_colour and bw-01.png">
```
Comment (greatislander) [inline components/Brand.vue]: Add alt tag.

Comment (greatislander) [inline components/Brand.vue]: Wrap both image and title in a link pointing to the home page ("/").

**Comment by TeddTech** [comment]
These changes have been completed

File: components/Brand.vue [hunk 3]
```diff
@@ -1,20 +1,8 @@
 <template>
 	<div class="brand">
-		<img src="~/static/img/We Count logos_colour and bw-01.png">
-		<h1> We Count </h1>
+		<nuxt-link to="/">
+			<img src="~/static/img/We Count logos_colour and bw-01.png" alt="Website Logo">
```
Comment (greatislander) [inline components/Brand.vue]: It's good to be a bit more specific -- an alt tag should describe the image. I would change this to "Logo for Project We Count"

Comment (TeddTech) [reply components/Brand.vue]: Okay, I have made the change

**Pull request merged**